plugins/nexus-repository-manager/src/api/nexus-repository-manager-api-client/nexus-repository-manager-api-client.test.ts

```diff
@@ -65,28 +65,42 @@ const handlers = [
     },
   ),
 
+  // Always returns manifest v2 schema 1, regardless of the accept header, to
+  // simulate a server that does not support schema 2.
```

Comment:
This has created a weird hybrid server that both does and doesn't support manifest schema 2 v2. It might be better to have two separate servers -- need to think on this.

Reply:
Tried having two separate servers, it was too much code & complexity for a somewhat-simple test, and seemed like it wasn't forward-thinking enough for when there's support for other formats. I'd suggest revisiting this after adding support for maven artifacts (next on our list).